code/create/dta_dicom2spec_rules.py

```diff
@@ -1,158 +1,115 @@
-## rules for converting the dta-dicoms into a bids-dataset.
-
+# rules for converting the dta-dicoms into a bids-dataset.
 from datalad_hirni.support.BIDS_helper import apply_bids_label_restrictions
 
 
 def _guess_modality(record, dcm_dict):
 
-    protocol = record.get("ProtocolName", None)
+    # not sure, if the strings in the dcm-headers are always consistent,
+    # so we just ignore the capital-letters.
+    protocol = record.get("ProtocolName", None).lower()
     if protocol:
 
-        # BEGIN Additional rule for forrest-structural
-        # TODO: Probably to be moved to some rule enhancement
-        if "VEN_BOLD" in protocol:
-            # TODO: Not clear yet; swi might be considered a datatype rather than
-            # a modality by respective BIDS extension:
-            # https://docs.google.com/document/d/1kyw9mGgacNqeMbp4xZet3RnDhcMmf4_BmRgKaOkO2Sc
-            return "swi"
-
-        if "DTI_" in protocol:
-            # TODO: What actually is the relevant part of protocol here?
+        if "dti_"   in protocol:
             return "dwi"
-
-        if "60dir".lower() in protocol.lower():
+        if "60dir"  in protocol:
             return "dwi"
-        if "120dir".lower() in protocol.lower():
+        if "120dir" in protocol:
```

Comment:
Just a coding-style suggestion. To have those three `if` statements works, of course. However, if you want to maintain code like that, there might be a need to enhance or change the list of things indicating a DWI. That would be easier to curate later on, if there was a respective actual list of such markers. Like:

```python
dwi_markers = ['dti_', '60dir', '120dir']
if any(m in protocol for m in dwi_markers):
    return "dwi"
```